Makefile

```diff
@@ -331,13 +331,19 @@ $(REPO_DIR)/$(REPO_ID)-repo/last-updated: $(CONF_DIR)/pkglist.$(REPO_ID) $(CONFI
 	@echo "Cleaning $(REPO_ID) yum repo, this could take a few minutes..."
 	$(VERBOSE)$(RM) -r $(REPO_DIR)/$(REPO_ID)-repo
 	@echo "Populating $(REPO_ID) yum repo, this could take a few minutes..."
+# TODO: This mess needs to be addressed using repoquery's relativepath tag.
 	@if [ ! -d $(REPO_PATH) ]; then echo -e "\nError yum repo path doesn't exist: $(REPO_PATH)\n"; exit 1; fi
 	$(call MKDIR,$(REPO_DIR)/$(REPO_ID)-repo)
 	$(VERBOSE)while read fil; do \
+		c=`tr [:upper:] [:lower:] <<< $$$${fil:0:1}`; \
 		if [ -f "$(REPO_PATH)/$$$$fil" ]; then \
-			 $(REPO_LINK) "$(REPO_PATH)/$$$$fil" $(REPO_DIR)/$(REPO_ID)-repo/$$$$fil; \
+			$(REPO_LINK) "$(REPO_PATH)/$$$$fil" $(REPO_DIR)/$(REPO_ID)-repo/$$$$fil; \
+		elif [ -f "$(REPO_PATH)/$$$${c}/$$$$fil" ]; then \
+			$(REPO_LINK) "$(REPO_PATH)/Packages/$$$${c}/$$$$fil" $(REPO_DIR)/$(REPO_ID)-repo/$$$$fil; \
```

Comment:
This tests for the existence of /foo/z/zed, but then symlinks to /foo/Packages/z/zed

Reply:
We don't have to check for the existence of repos/<reponame>-repo/Packages/z/ or repos/<reponame>-repo/z/ while doing the symlinking as a result. Do you think the difference in dir structure may confuse the user? Or perhaps the same package name may pop up in multiple source directories?

Reply:
No I think these changes are useful to the average user.  What I was trying to say was that there is a bug on line 342.  Line 342 is identical to line 346, but it should not be.  Line 341 checks for the existence of a package but then like 342 symlinks to a different path that probably does not exist.  Line 342 just needs the "Packages" portion of the path removed.

Reply:
Ah OK should be fixed now.